atuin-client/config.toml

```diff
@@ -73,3 +73,13 @@
 #   "^secret-cmd",
 #   "^innocuous-cmd .*--secret=.+"
 # ]
+
+## should env-vars be captured in the history
+# capture_env = false
+
+## prevent env-vars matching any of these regexes from being written to history.
+## As with history_filter, these regexes are unanchored.
+# env_filter = [
+#   "^SECRET_.*",
+#   "^AWS_SECRET_ACCESS_KEY$"
+# ]
```

Comment:
Allow capturing env-variables.
`false` by default, as there are frequently some secret-like values stored in env, so not everyone might be comfortable discovering that they are recorded by default.

Also allow to configure black list of vars.

Reply:
Maybe it should be white list?